.circleci/config.yml

```diff
@@ -48,6 +48,23 @@ executors:
       GRADLE_OPTS: "-Dorg.gradle.daemon=false -Dorg.gradle.workers.max=2"
 
 commands:
+  install-ruby:
+    description: Install Ruby
+    steps:
+      - run:
+          name: install chruby
+          command: |
+            wget -O chruby-0.3.9.tar.gz https://github.com/postmodern/chruby/archive/v0.3.9.tar.gz
+            tar -xzvf chruby-0.3.9.tar.gz
+            cd chruby-0.3.9/
+            sudo make install
+      - run: ruby-install ruby 2.7.1
+      - run:
+          name: Set chruby
+          command: |
+            echo 'source /usr/local/share/chruby/chruby.sh' >> $BASH_ENV
+            echo 'chruby ruby-2.7.1' >> $BASH_ENV 
```

Comment:
This is not ideal because of install time but is needed because `octokit` requires >= Ruby 2.7 and `octokit` can't be locked in a `4.x.x` otherwise we would need to downgrade `danger` from `9.x.x` to `8.x.x` 😬 

I tried using Circle's Ruby Orb that uses `rvm` but that was failing to install binaries because of `apt-get` issues

This works "for now" and is only being merged into `bc5-support` branch. I will fix this later on 😇